### PR TITLE
Upper bounds for final Julia 0.6-compatible release

### PR DESCRIPTION
**Pull request by lbenet**
This should make easier to upgrade for compatibility with Julia 0.7 and 1.0

### CODE REVIEW

**Comment by lbenet** [comment]
@dpsanders I think this is ready to avoid breaking changes in our packages. Do you have any comments/suggestions?

File: REQUIRE [hunk 1]
```diff
@@ -1,7 +1,8 @@
 julia 0.6
 Reexport 0.1.0
 
-IntervalArithmetic 0.13.0
-IntervalRootFinding 0.2.0
-IntervalContractors 0.2.0
+IntervalArithmetic 0.13.0 0.14.0
```
[the remark this answers is not in the document]
Comment (lbenet) [reply REQUIRE]: I agree with the comment. Yet, i am not too sure that the changes introduced in 0.14.0 break something else. I can certainly change that, but I can't add 0.15.0, unless that version is tagged. (I thought that was ok, but was having problems with the tests, even locally.) Should I delete then the upper bound for IntervalArithmetic?

**Comment by dpsanders** [comment]
Actually, isn't master of IntervalRootFinding compatible with v0.14? In which case we should just tag a new version there.

File: REQUIRE (hunk 1 above)
[the remark this answers is not in the document]
Comment (lbenet) [reply REQUIRE]: Current master, of IntervalRootFinding is compatible (tests pass locally) with v0.14.0 of IntervalArithmetic. Yet, v0.2.1 is not compatible with that version (tests fail); v0.2.0 is compatible with v0.13.0. So I think the entry is correct.

**Comment by lbenet** [comment]
I'll implement your other suggestions, though I am not completely sure that they will work...

**Comment by lbenet** [comment]
Done. Let's see how travis reacts.

**Comment by lbenet** [comment]
Tests pass!

**Comment by dpsanders** [comment]
I have added an upper bound for Julia. I think this is ready to merge.

**Comment by lbenet** [comment]
Merging! Thanks a lot!

**Pull request merged**